docs/docs-beta/docs/guides/deploy/deployment-options/gcp.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Deploying Dagster to Google Cloud Platform"
+description: To deploy Dagster to GCP, Google Compute Engine (GCE) can host the Dagster webserver, Google Cloud SQL can store runs and events, and Google Cloud Storage (GCS) can act as an IO manager.
+sidebar_position: 60
+---
+
+To deploy Dagster to GCP, Google Compute Engine (GCE) can host the Dagster webserver, Google Cloud SQL can store runs and events, and Google Cloud Storage (GCS) can act as an IO manager.
+
+## Hosting the Dagster webserver or Dagster Daemon on GCE
+
+To host the Dagster webserver or Dagster daemon on a bare VM or in Docker on GCE, see [Running Dagster as a service](/guides/deploy/deployment-options/deploying-dagster-as-a-service).
+
+## Using Cloud SQL for run and event log storage
+
+We recommend launching a Cloud SQL PostgreSQL instance for run and events data. You can configure the webserver to use Cloud SQL to run and events data by setting blocks in your `$DAGSTER_HOME/dagster.yaml` appropriately:
+
+```python file=/deploying/dagster-pg.yaml
+storage:
+  postgres:
+    postgres_db:
+      username: my_username
+      password: my_password
+      hostname: my_hostname
+      db_name: my_database
+      port: 5432
+```
+
+In this case, you'll want to ensure you provide the right connection strings for your Cloud SQL instance, and that the node or container hosting the webserver is able to connect to Cloud SQL.
+
+Be sure that this file is present, and _DAGSTER_HOME_ is set, on the node where the webserver is running.
```

Comment:
```suggestion
Be sure that this file is present, and `_DAGSTER_HOME_` is set, on the node where the webserver is running.
```

Reply:
Fixed in https://github.com/dagster-io/dagster/pull/27087/commits/2c1ce41a43176fa33289d68ab2e8d5acfe6455d6